src/components/LeftSidebar/LeftSidebar.jsx

```diff
@@ -0,0 +1,167 @@
+import style from "./LeftSidebar.module.css";
+import user from "../../Images/user.jpg";
+import friend1 from "../../Images/friend1.jpg";
+import friend2 from "../../Images/friend2.jpg";
+import friend3 from "../../Images/friend3.jpg";
+
+function LeftSidebar() {
+  return (
+    <>
+      <div className={style.Sidebar}>
+        <div className={style.imgIcons}>
+          <div className={style.image}>
+            <figure>
+            <img src={user} alt="" />
+            </figure>
+          </div>
+          <div className={style.icons}>
+            <div className={style.gridIcon}>
+              <svg
```

Comment:
refactor to a separate component

Reply:
I didn't get it. is that what u talking about ?->https://dev.to/michaellarocca/a-guide-to-better-code-organization-in-react-through-file-separation-4gog

Reply:
or did you mean about the name of classes I should do it like this grid-icon instead of gridIcon?

Reply:
Or formatting the document?

Reply:
<SVG props/>